clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -66,6 +72,7 @@ export async function activate(
   context: vscode.ExtensionContext,
 ): Promise<__ExtensionApi & __AnalysisApi> {
   DialectRegistry.clear();
+  Utils.setExtensionFolder(context.extensionPath);
```

Comment:
This should be `context.globalStorageUri`

Reply:
resolved at : c64a4ff4ba28fa10c02c5e915812658e86175bc3